src/Model/SmimeSigningCertificate.php

```diff
@@ -123,14 +123,14 @@ public function getCMSFields(): FieldList
             [
                 UploadField::create('SigningCertificate', 'Certificate File')
                     ->setFolderName(self::$uploadFolder)
-                    ->setAllowedExtensions(['crt'])
-                    ->setDescription('Upload a valid <strong>.crt</strong> file for this email address. '
+                    ->setAllowedExtensions(['pem'])
```

Comment:
This has been updated since it now requires `.pem` files 
![Screen Shot 2024-06-11 at 10 47 32 AM](https://github.com/silverstripeltd/user-configurable-smime-forms/assets/27745093/afe214eb-1650-4afb-bffa-84834f740a4d)